[0m[[0m[31merror[0m] [0m[0m/home/ubuntu/report/src/main/scala/report.scala:71:1: ';' expected but '#' found.[0m
[0m[[0m[31merror[0m] [0m[0m#val df_transform = df_complete.withColumn("start",to_timestamp($"start","yyyy-MM-ddThh:mm:ss.SSS"))[0m
[0m[[0m[31merror[0m] [0m[0m^[0m
[0m[[0m[31merror[0m] [0m[0mone error found[0m
